src/tutorials/0005-regular-files-api/03.js

```diff
@@ -49,7 +49,12 @@ const validate = async (result, ipfs) => {
   if (JSON.stringify(expectedResult) === JSON.stringify(result)) {
     return {
       success: utils.validationMessages.SUCCESS,
-      logDesc: 'Your `add` command returned the array of objects below. Notice in particular the `cid` ' + valueText + ", since we'll need " + thatText + ' to access ' + fileText + ' again later. The `path` matches the `cid` for ' + fileText + ", but we'll see in future lessons that that's not always true.",
+      logDesc: [
+        'The output is very long because the CID is represented as an `Object` internally.',
```

Comment:
```suggestion
        'Your `add` command returned the array of objects below. The output is very long because the CID is represented as an `Object` internally, but if you scroll down we'll offer you a more condensed view.',
```